cli/commands/queueWithdrawal.go

```diff
@@ -0,0 +1,105 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/Layr-Labs/eigenlayer-contracts/pkg/bindings/EigenPod"
+	"github.com/Layr-Labs/eigenlayer-contracts/pkg/bindings/IDelegationManager"
+	"github.com/Layr-Labs/eigenpod-proofs-generation/cli/core"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+type TQueueWithdrawallArgs struct {
+	EthNode     string
+	EigenPod    string
+	Sender      string
+	EstimateGas bool
+}
+
+func QueueWithdrawalCommand(args TQueueWithdrawallArgs) error {
+	ctx := context.Background()
+
+	isSimulation := args.EstimateGas
+
+	eth, err := ethclient.DialContext(ctx, args.EthNode)
+	core.PanicOnError("failed to reach eth node", err)
+
+	chainId, err := eth.ChainID(ctx)
+	core.PanicOnError("failed to load chainId", err)
+
+	acc, err := core.PrepareAccount(&args.Sender, chainId, args.EstimateGas)
+	core.PanicOnError("failed to parse private key", err)
+
+	dm, err := IDelegationManager.NewIDelegationManager(DelegationManager(chainId), eth)
+	core.PanicOnError("failed to reach delegation manager", err)
+
+	pod, err := EigenPod.NewEigenPod(common.HexToAddress(args.EigenPod), eth)
+	core.PanicOnError("failed to reach eigenpod", err)
+
+	_reg, err := pod.WithdrawableRestakedExecutionLayerGwei(nil)
+	core.PanicOnError("failed to load REG", err)
+	reg := new(big.Int).SetUint64(_reg)
+
+	podOwner, err := pod.PodOwner(nil)
+	core.PanicOnError("failed to read podOwner", err)
+
+	res, err := dm.GetWithdrawableShares(nil, podOwner, []common.Address{core.BeaconStrategy()})
+	core.PanicOnError("failed to read beacon strategy withdrawable shares", err)
+
+	_depositShares, err := dm.ConvertToDepositShares(nil, podOwner, []common.Address{core.BeaconStrategy()}, []*big.Int{res.WithdrawableShares[0]})
```

Comment:
this should accept a flag for the amount of withdrawableShares to withdraw. this will withdraw all shares no matter what